website/blog/OneTable-is-now-Apache-XTable.md

```diff
@@ -0,0 +1,67 @@
+[growing interest](https://x.com/OnetableOSS/status/1725685237215322294?s=20). Additionally, XTable has a diverse set 
+of contributors from organizations such as Microsoft, Onehouse, Google, Walmart, Adobe, Cloudera, and Dremio. With the 
+Apache incubation, we are also looking forward to connecting with more contributors who have similar interests in this space.
+
+Incubating XTable at the Apache Software Foundation (ASF) signifies more than just the technical evolution of a project; 
+it's a step towards fostering a vibrant community around it. This milestone is exciting for several reasons. Firstly, 
+the ASF's incubation process is renowned for its emphasis on community development and open governance, aligning 
+perfectly with our vision for XTable. By embracing the Apache way, we're not just contributing code; we're building a 
+foundation for a project that's open, inclusive, and sustainable. The incubation phase will equip XTable with the 
+mentoring and services necessary to thrive as a Top Level Project (TLP), ensuring it benefits from the ASF's vendor-neutral 
+environment.
+
+This environment encourages innovation, collaboration, and the sharing of ideas, making it an ideal breeding ground for 
+XTable to grow. Moreover, the ASF's global reach and reputation will enhance XTable's visibility and adoption, creating 
+new opportunities for collaboration with other projects and communities. We're excited about the journey ahead and the 
+potential for XTable to become a key part of the ASF ecosystem, contributing to its mission of providing software for 
+the public good.
+
+<img src="/images/blog/XTable/xtable-docs.png" alt="drawing" style={{width:'80%', display:'block', marginLeft:'auto', 
+marginRight:'auto', marginTop:'18pt', marginBottom:'18pt'}} />
+
+For those interested in exploring Apache XTable, the official website is a good starting point. The documentation 
+section hosts a great hands-on [quickstart](https://onetable.dev/docs/how-to) guide to getting acquainted with XTable, 
+providing a straightforward way to experience its interoperability capabilities firsthand. If you have specific ideas, 
+questions, or seek direct interaction, the [discussions](https://github.com/onetable-io/onetable/discussions) section 
+is available for more in-depth exchanges. We invite you to contribute to the project by submitting pull requests or 
+filling issues to the [XTable GitHub repository](https://github.com/apache/incubator-xtable). Contributions in the early 
+phase of the project are going to be critical as we build XTable together. Joining the 
+[XTable mailing list](mailto:dev-subscribe@xtable.apache.org) is another excellent way to stay informed and engage with 
+the project. We are really excited about the future of Apache XTable and building it together with the vibrant data community.
+
+Follow Apache XTable on [LinkedIn](https://www.linkedin.com/company/onetable-oss/) and [Twitter](https://twitter.com/OnetableOSS) 
```

Comment:
lets rename these as well? before we push the blog out?

Reply:
ya probably need the links right from beginning here. I just went and updated the social accounts and these links + all other references on site